config/routes.rb

```diff
@@ -53,6 +53,8 @@
   match 'home' => 'home#front'
   resources :relationships, only: [:create, :destroy]
 
+  get '/wiki/:id/comments', to: 'wiki#wiki_comments'
```

Comment:
This `wiki#wiki_comments` seems redundant - -any problem using just `wiki#comments` and action name just `comments`? sorry if I miss something here.